### ISSUE TITLE
Image Folder setting doesn't work

### ISSUE THREAD

**Issue by xdaanx** [NaturalDocs/NaturalDocs#65]
I have set an image folder, but it doesn't work.

Is my understanding correct, that if I have set 'Image Folder', images should be visible from all directories (architecture and sources) without specifying the path? 

```
Source Folder: ../sources/
   Name: Sources
Source Folder 2: ../architecture
   Name: Architecture
Image Folder: ../architecture/img
```

From architecture folder, I am able to use relative path: (see img/image.png), but (see image.png) works only from architecture/img. 

It is the same also for command line -img parameter.

Natural Docs: **2.1.1** and also **2.2 development release 3**
Mono: 4.8.1

My directory structure:
```
docs
+-- architecture
    +-- img
+-- sources
+-- config
+-- html
+-- Makefile
```

**Comment by NaturalDocs** [comment]
I think I found the bug.  Try this version:
https://www.naturaldocs.org/download/temp/Natural_Docs_temp5.zip

**Comment by xdaanx** [comment]
It works. Thank you.

**Comment by NaturalDocs** [comment]
This fix has been included in 2.2 Development Release 4.

**Comment by NaturalDocs** [comment]
[This fix has also been included in 2.2.](https://www.naturaldocs.org/download/)
